cmd/image-loader/main.go

```diff
@@ -65,28 +68,32 @@ var (
 )
 
 type opts struct {
-	versionsFile  string
-	versionFilter string
-	registry      string
-	dryRun        bool
-	addonsPath    string
+	configurationFile string
+	versionsFile      string
+	versionFilter     string
+	registry          string
+	dryRun            bool
+	addonsPath        string
 }
 
 func main() {
-	klog.InitFlags(nil)
+	var err error
 
 	logOpts := kubermaticlog.NewDefaultOptions()
+	logOpts.Format = kubermaticlog.FormatConsole
 	logOpts.AddFlags(flag.CommandLine)
 
 	o := opts{}
-	flag.StringVar(&o.versionsFile, "versions", "charts/kubermatic/static/master/versions.yaml", "The versions.yaml file path")
+	flag.StringVar(&o.configurationFile, "configuration-file", "", "Path to the KubermaticConfiguration YAML file")
```

Comment:
I would mention that the two flags are mutually (i.e. `configuration-file` and `versions-file`) exclusive and I would add some input validation. If both are specified or none of them I would fail immediately.

Reply:
Done.